encoding/dot/decode_test.go

```diff
@@ -61,7 +51,7 @@ func TestRoundTrip(t *testing.T) {
 	}
 }
 
-const directed = `digraph G {
+const directed = `digraph {
```

Comment:
We can add supporting methods at a later stage if people ask for them. However, I think the labels here should be put back, purely to demonstrate that your code here does handle (though lossily) DOT with labels.